hc/api/tests/test_sendalerts.py

```diff
@@ -26,17 +26,16 @@ def test_it_handles_few(self, mock):
         handled_names = []
         for args, kwargs in mock.call_args_list:
             handled_names.append(args[0].name)
-
+            handled_names.sort
```

Comment:
This functions seems like its 0(N) 2 .. there should be better way of handling this.. no?